packages/astro/src/core/app/types.ts

```diff
@@ -37,7 +37,7 @@ export type SSRManifest = {
 	adapterName: string;
 	routes: RouteInfo[];
 	site?: string;
-	base?: string;
+	base: string;
```

Comment:
I traced this type, and confirmed that `base` is always created by the function `buildManifest`